payroll/models/hr_payslip.py

```diff
@@ -236,6 +240,7 @@ class HrPayslip(models.Model):
         string="Payslip Batches",
         readonly=True,
         copy=False,
+        tracking=1,
```

Comment:
This is a one2many field and there are potentially many detail lines per payslip multiplied by the number of payslips. Is there any performance implications of adding this? If there aren't that's fine, but if there are then is there any benefit to tracking this field?

Reply:
Actually is a many2one field, not a one2many. 

The tracking in this field only reports the change in run_id field, it don't retrieve all the run_id related records. 

Tracking this is useful because you can keep track of someone accidentally links a payslip in an incorrect run_id. 

This only applies to the records modified inside the payslip form. When you crate payslips from Payslip Batches this is not tracked.

Reply:
Oops, sorry. My bad. I was looking at it in the web interface and it looked like you were tracking details_by_salary_rule_category.

Reply:
`tracking=True` please